docs/install/build_from_source.md

```diff
@@ -226,109 +148,62 @@ To build OpenCV from source code, you need the [cmake](https://cmake.org) librar
    sudo make install
    ```
 
-4. Add the lib path to your configuration such as `~/.bashrc`.
+* Add the lib path to your configuration such as `~/.bashrc`.
 
    ```bash
    export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/usr/local/lib/pkgconfig/
    ```
 
-</div>
-</div>
-
-<div class="windows">
-
-First download and install [OpenCV](http://opencv.org/releases.html), then set
-the environment variable `OpenCV_DIR` to point to the OpenCV build directory.
-
-</div>
-
-#### Optional: [CUDA](https://developer.nvidia.com/cuda-downloads)/[cuDNN](https://developer.nvidia.com/cudnn) for Nvidia GPUs
-
-MXNet is compatible with both CUDA 7.5 and 8.0. It is recommended to use cuDNN 5.
-
-<div class="linux">
-<div class="ubuntu">
-
-Install CUDA 7.5 and cuDNN 5 on Ubuntu 14.04
-
-```bash
-wget http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1404/x86_64/cuda-repo-ubuntu1404_7.5-18_amd64.deb
-sudo dpkg -i cuda-repo-ubuntu1404_7.5-18_amd64.deb
-echo "deb http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1404/x86_64 /" | sudo tee /etc/apt/sources.list.d/nvidia-ml.list
-sudo apt-get update
-sudo apt-get install -y linux-image-extra-`uname -r` linux-headers-`uname -r` linux-image-`uname -r`
-sudo apt-get install -y cuda libcudnn5-dev=5.0.5-1+cuda7.5
-```
-
-</div>
-</div>
 
 ### Build
 
-<div class="linux macos">
-
-First clone the recent codes
-
+1. Clone the MXNet project.
 ```bash
-git clone --recursive https://github.com/dmlc/mxnet
+git clone --recursive https://github.com/apache/incubator-mxnet mxnet
 cd mxnet
 ```
 
-File
-[`make/config.mk`](https://github.com/dmlc/mxnet/blob/master/make/config.mk)
-contains all the compilation options. You can edit it and then `make`. There are
-some example build options
-
-If you want to build MXNet with C++ language binding, please make sure you read [Build the C++ package](#build-the-c-package) first.
-
-</div>
+There is a configuration file for make,
+[`make/config.mk`](https://github.com/apache/incubator-mxnet/blob/master/make/config.mk), that contains all the compilation options. You can edit it and then run `make`.
 
-<div class="linux">
+To enable C++ package, just add `USE_CPP_PACKAGE=1` when you run `make`.
 
-- Build without using OpenCV. `-j` runs multiple jobs against multi-core CPUs.
+Other typical configurations are:
 
-  ```bash
-  make -j USE_OPENCV=0
-  ```
+* `-j` runs multiple jobs against multi-core CPUs. Example using all cores on Linux:
 
-- Build with both GPU and OpenCV support
-
-  ```bash
-  make -j USE_BLAS=openblas USE_CUDA=1 USE_CUDA_PATH=/usr/local/cuda USE_CUDNN=1
-  ```
-
-</div>
-
-<div class="macos">
-
-- Build with the default BLAS library and clang installed with `xcode` (OPENMP
-  is disabled because it is not supported in default by clang).
+```bash
+make -j$(nproc)
+```
 
-  ```bash
-  make -j USE_BLAS=apple USE_OPENCV=0 USE_OPENMP=0
-  ```
+* Build without using OpenCV:
 
-</div>
+```bash
+make USE_OPENCV=0
```

Comment:
Maybe this would be a good place to put the BLAS libraries explanation from https://github.com/apache/incubator-mxnet/pull/11148. What do you think?

Reply:
This caused me to rearrange the page for a better flow. This is all good info. It should be double-checked in case it is only for cmake and won't work for make.

Reply:
Ah, yes you're right